vendor/knative.dev/networking/pkg/http/header/header.go

```diff
@@ -78,6 +78,9 @@ const (
 	// load balancers to not load balance the respective request but to
 	// send it to the request's target directly.
 	PassthroughLoadbalancingKey = "K-Passthrough-Lb"
+
+	// IgnoreMetricsKey is used to mark requests that should not be counted in metrics.
+	IgnoreMetricsKey = "K-Ignore-Metrics"
```

Comment:
You can test this in this PR but has to be moved to the related repo: https://github.com/knative/networking

Reply:
PR created: https://github.com/knative/networking/pull/1025